samples/subsys/settings/native_posix.overlay

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
I do not think this file is needed anymore, I have been able to run this sample without it. Can you check it?